images/fedora/f37/Containerfile

```diff
@@ -13,6 +13,8 @@ COPY README.md /
 
 RUN sed -i '/tsflags=nodocs/d' /etc/dnf/dnf.conf
 RUN dnf -y swap coreutils-single coreutils-full
+RUN dnf -y swap glibc-minimal-langpack glibc-all-langpacks
+RUN rm /etc/rpm/macros.image-language.conf
```

Comment:
It will be better to make changes to the RPM configuration before running DNF.  Otherwise, those changes won't affect the DNF invocations.

I don't know if it actually makes a difference in this case, but it's better to be safe than sorry.  :)

Reply:
I took the liberty to change this in https://github.com/containers/toolbox/pull/1225